apps/mobile/package-lock.json

```diff
@@ -11,9 +11,6 @@
         "native/",
```

Comment:
There shouldn't be changes here as you didn't add any change in original `package.json`file

Reply:
Agreed